packages/a11y/tsconfig.json

```diff
@@ -1,10 +1,5 @@
 {
 	"$schema": "https://json.schemastore.org/tsconfig.json",
```

Comment:
While we are here, why do we need this "$schema" in ever tsconfig file? Don't the IDEs and TS automatically infer that?

Reply:
I see that it was added in #59861

Reply:
Some might infer it based on the filename, but including it ensures that supporting environments will use the appropriate schema. There's little downside to including it and the benefits are significant.